independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/update/rewrite/QuarkusUpdates.java

```diff
@@ -48,6 +52,24 @@ public static FetchResult createRecipe(MessageWriter log, Path target, MavenArti
                 break;
         }
 
+        for (List<ExtensionUpdateInfo> nonPlatformExtensionsUpdates : request.projectExtensionsUpdateInfo
+                .getNonPlatformExtensions().values()) {
+            for (ExtensionUpdateInfo nonPlatformExtensionsUpdate : nonPlatformExtensionsUpdates) {
+                if (nonPlatformExtensionsUpdate.getCurrentDep().isPlatformExtension()) {
+                    // add, my understanding is that we should define the version? As a dependency, as a managed one?
+                    // not completely sure how to make it work for a multi-module project?
```

Comment:
Should we log something in this case? AFAIU, we are not applying any update in this case.